pkg/confgen/grafana_jsonnet.go

```diff
@@ -292,3 +327,94 @@ func (cg *ConfGen) addPanelsToDashboards(dashboards Dashboards) (Dashboards, err
 
 	return dashboards, nil
 }
+
+const grafanaDirPath = "grafana/grafonnet-lib/grafonnet"
+
+//go:embed grafana/grafonnet-lib/grafonnet
+var grafanaDir embed.FS
+
+type embedImporter struct {
+	fsBase  embed.FS
+	fsCache map[string]*fsCacheEntry
+}
+
+type fsCacheEntry struct {
+	contents *[]byte
+	exists   bool
+}
+
+func (cg *ConfGen) GenerateGrafanaJson() (string, error) {
```

Comment:
It seems that `GenerateGrafanaJson()` is called only in `confgen_test.go` and not in production code. If it is still needed, maybe it can be moved to `confgen_test.go`?

Reply:
GenerateGrafanaJson() is the function that will be called from the Network Observability Operator to create the dashboards for Openshift.